web/tests/test_chapters.py

```
@@ -0,0 +1,247 @@
+            "description": "A new test event description",
+            "event_type": "workshop",
+            "start_time": tomorrow.strftime("%Y-%m-%d %H:%M:%S"),
+            "end_time": tomorrow_plus_2.strftime("%Y-%m-%d %H:%M:%S"),
+            "location": "New Location",
+            "is_public": "on",
+        }
+
+        response = self.client.post(reverse("create_chapter_event", args=[self.chapter.slug]), event_data)
+
+        # Check that the event was created
+        self.assertTrue(ChapterEvent.objects.filter(title="New Test Event").exists())
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertTrue`

Replace `assertTrue(...)` with `assert ...`

(PT009)